composer.json

```diff
@@ -44,25 +45,25 @@
 			"@qa-tester"
 		],
 		"qa-codesniffer": [
```

Comment:
Could you please drop scripts a introduce Makefile? Just copy paste from here Could you please switch it to Github Actions? Just copy paste whole .github folder from here https://github.com/contributte/bare/.